packages/roosterjs-plugin-picker/lib/PickerPlugin.ts

```diff
@@ -260,8 +275,12 @@ export default class PickerPlugin<T extends PickerDataProvider = PickerDataProvi
                     wordBeforeCursor[0] == this.pickerOptions.triggerCharacter
                 ) {
                     this.setIsSuggesting(true);
-                    let shortWord = wordBeforeCursor.substring(1).trim();
-                    this.dataProvider.queryStringUpdated(shortWord);
+                    const wordBeforeCursorWithoutTriggerChar = wordBeforeCursor.substring(1);
+                    let trimmedWordBeforeCursor = wordBeforeCursorWithoutTriggerChar.trim();

```

Comment:
Wish these two lines didn't have to be repeated, but code looks good to me otherwise